manifests/charts/aperture-agent/templates/rbac.yaml

```diff
@@ -165,6 +165,19 @@ rules:
   - patch
   - update
   - watch
+# auto scale permissions
+- apiGroups: ["*"]
+  resources: ["*"]
```

Comment:
Is this intentional ? I believe the agent is expected to function with limited permissions. 
Having * would mean read access to all the resources.
Usually secrets and configmaps are let out

Reply:
We are getting read permissions to all objects in order to auto-discover any object that is scalable. Can keep this permission under Kubernetes control point discovery helm flag, cc: @hdkshingala